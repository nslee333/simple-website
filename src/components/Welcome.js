import React from "react";
import {StyledWelcome} from "../UI/Common";

export default function Welcome () {
  return (
    <>
      <StyledWelcome>
        <div className='StyledWelcome__Div'>
          <h4>BendJS</h4>
          <h5>A Javascript meetup in Bend, Oregon</h5>
          <h5>
          This group is for anyone interested in learning Javascript, 
          sharing something they know, networking with other devs, and having a great time! 
          </h5>
          <h5>
          All levels are welcome. Please join our slack workspace for future updates and continuing the conversation.
          </h5>
        </div>
      </StyledWelcome>
    </>
  )
}