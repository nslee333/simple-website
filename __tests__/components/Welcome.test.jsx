import { render } from "@testing-library/react";
import Welcome from "../../src/components/Welcome";
import renderer from "react-test-renderer";

test("Welcome renders without crashing", () => {
  render(<Welcome />);
});

test("Welcome renders correctly according to snapshot.", () => {
  const tree = renderer.create(<Welcome />).toJSON();

  expect(tree).toMatchSnapshot();
});
