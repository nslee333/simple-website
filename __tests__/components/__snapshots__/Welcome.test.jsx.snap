// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome renders correctly according to snapshot. 1`] = `
<div
  className="flex justify-center px-5 pt-2"
>
  <div
    className="w-[50rem] italic text-white xs:mx-2 xs:my-8 sm:my-16 md:my-32"
  >
    <h4
      className="my-5 xs:text-3xl md:text-4xl"
    >
      BendJS
    </h4>
    <h5
      className="my-5 xs:text-2xl md:text-3xl"
    >
      A Javascript meetup in Bend, Oregon
    </h5>
    <h5
      className="my-10 xs:text-2xl md:text-3xl"
    >
      This group is for anyone interested in learning Javascript, sharing something they know, networking with other devs, and having a great time!
    </h5>
    <h5
      className="xs:text-2xl sm:my-5 md:my-10 md:text-3xl"
    >
      All levels are welcome. Please join our slack workspace for future updates and continuing the conversation.
    </h5>
  </div>
</div>
`;
